doc/source/docs/reference/sindex.rst

```diff
@@ -4,9 +4,9 @@ Spatial index
 .. currentmodule:: geopandas
 
 GeoPandas offers built-in support for spatial indexing using an R-Tree algorithm.
-Depending on the ability to import ``pygeos``, GeoPandas will either use
+Depending on the ability to import PyGEOS, GeoPandas will either use
 ``pygeos.STRtree`` or ``rtree.index.Index``. The main interface for both is the
-same and follows the ``pygeos`` model.
+same and follows the PyGEOS model.
```

Comment:
This is actually out of date with shapely 2.0... (no need to act on that, it just a note for myself)